doc/htmldoc/examples/index.rst

```diff
@@ -196,6 +196,13 @@ PyNEST examples
            * :doc:`../auto_examples/evaluate_tsodyks2_synapse`
 
 
+.. grid:: 1 1 2 3
+
+    .. grid-item-card:: E-prop plasticity
+           :img-top: ../static/img/pynest/eprop_supervised_classification_infrastructure.png
+
+           * :doc:`../auto_examples/eprop_plasticity/index`
```

Comment:
```suggestion
  * :doc:`/auto_examples/eprop_plasticity/eprop_supervised_classification_evidence-accumulation`
  * :doc:`/auto_examples/eprop_plasticity/eprop_supervised_regression_sine-waves`
  * :doc:`/auto_examples/eprop_plasticity/eprop_supervised_regression_handwriting`
  * :doc:`/auto_examples/eprop_plasticity/eprop_supervised_regression_infinite-loop`
```

Reply:
@jessica-mitchell Would it be okay here to also turn the header of this section into a clickable link to the index so that the user can also see the background information of the tasks?

```rst
.. grid:: 1 1 2 3

    .. grid-item-card:: :doc:`../auto_examples/eprop_plasticity/index`
           :img-top: ../static/img/pynest/eprop_supervised_classification_infrastructure.png

           * :doc:`/auto_examples/eprop_plasticity/eprop_supervised_classification_evidence-accumulation`
           * :doc:`/auto_examples/eprop_plasticity/eprop_supervised_regression_sine-waves`
           * :doc:`/auto_examples/eprop_plasticity/eprop_supervised_regression_handwriting`
           * :doc:`/auto_examples/eprop_plasticity/eprop_supervised_regression_infinite-loop`
```

Reply:
@jessica-mitchell Do you maybe know why, during the compilation of the docs, the warning
```
nest-simulator/doc/htmldoc/auto_examples/eprop_plasticity/index.rst: WARNING: document isn't included in any toctree
```
appears, despite the index being linked here?

Reply:
this is because the index page is not in a toctree directive

> @jessica-mitchell Would it be okay here to also turn the header of this section into a clickable link to the index so that the user can also see the background information of the tasks?
> 
> ```
> .. grid:: 1 1 2 3
> 
>     .. grid-item-card:: :doc:`../auto_examples/eprop_plasticity/index`
>            :img-top: ../static/img/pynest/eprop_supervised_classification_infrastructure.png
> 
>            * :doc:`/auto_examples/eprop_plasticity/eprop_supervised_classification_evidence-accumulation`
>            * :doc:`/auto_examples/eprop_plasticity/eprop_supervised_regression_sine-waves`
>            * :doc:`/auto_examples/eprop_plasticity/eprop_supervised_regression_handwriting`
>            * :doc:`/auto_examples/eprop_plasticity/eprop_supervised_regression_infinite-loop`
> ```

Ah this is a nice solution, imo.

Reply:
> @jessica-mitchell Do you maybe know why, during the compilation of the docs, the warning
> 
> ```
> nest-simulator/doc/htmldoc/auto_examples/eprop_plasticity/index.rst: WARNING: document isn't included in any toctree
> ```
> 
> appears, despite the index being linked here?
 
This is because  it's not in the `toctree` at the bottom of the page. You can just add it, and it won't complain anymore. Sphinx expects all documents to be included in a toctree.

Reply:
Both fixed in https://github.com/nest/nest-simulator/pull/2867/commits/efab736686e042fb15c147115f0ab5c042738573